doc/source/user_guide/groupby.rst

```diff
@@ -87,11 +87,10 @@ The mapping can be specified many different ways:
 * A Python function, to be called on each of the axis labels.
 * A list or NumPy array of the same length as the selected axis.
 * A dict or ``Series``, providing a ``label -> group name`` mapping.
-* For ``DataFrame`` objects, a string indicating a column to be used to group.
-  Of course ``df.groupby('A')`` is just syntactic sugar for
-  ``df.groupby(df['A'])``, but it makes life simpler.
-* For ``DataFrame`` objects, a string indicating an index level to be used to
-  group.
+* For ``DataFrame`` objects, a string indicating either a column name or
+an index level name to be used to group.
+* df.groupby('A') is just syntactic sugar for df.groupby(df['A']),
```

Comment:
You have to put your code statements in backticks like it was done previously

Reply:
Backticks added... sorry for such silly mistakes @phofl